trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py

```diff
@@ -318,12 +318,15 @@ def calculate_lift_vortex_lattice(conditions,settings,geometry):
 
     # iterate over wings
     total_lift_coeff = 0.0
+    AR               = 0.0
     wing_lifts = Data()
 
     for wing in geometry.wings.values():
 
-        [wing_lift_coeff,wing_drag_coeff] = weissinger_vortex_lattice(conditions,settings,wing)
+        [wing_lift_coeff,wing_drag_coeff,wing_AR] = weissinger_vortex_lattice(conditions,settings,wing)
         total_lift_coeff += wing_lift_coeff * wing.areas.reference / vehicle_reference_area
-        wing_lifts[wing.tag] = wing_lift_coeff
-
-    return total_lift_coeff, wing_lifts
+        AR += wing_AR * wing.areas.reference / vehicle_reference_area
```

Comment:
Wait, I don't know that this assumption quite holds. I think it should be a weighted average. But you have a summation.

If I keep adding many low aspect ratio wings I don't get a high aspect ratio wing (unless they're tip to tip).